client/src/locales/fr-FR.json

```diff
@@ -1347,22 +1347,51 @@
         },
         "paymentMethodsPage": {
             "title": "Votre moyen de paiement",
-            "description": "Le paiement s’effectue automatiquement par votre carte bancaire.",
-            "addCard": "Ajouter",
-            "addPaymentMethodFailed": "Impossible d'enregistrer cette carte.",
-            "addPaymentMethodSuccessSetDefaultFailed": "La carte a été ajoutée avec succès mais n'a pas pu être définie comme moyen de paiement par défaut.",
-            "setDefaultPaymentMethodFailed": "Impossible d'utiliser cette carte comme votre moyen de paiement par défaut.",
-            "addPaymentMethodTitle": "Ajouter une carte bancaire",
-            "addPaymentMethodConfirm": "Ajouter cette carte",
+            "description": "Le paiement s'effectue automatiquement par votre carte bancaire.",
             "useAsDefault": "Définir comme moyen de paiement actif",
-            "activeMethodLabel": "Active",
+            "activeMethodLabel": "active",
+            "activeDefault": "Carte utilisée par défaut",
             "retrieveFailed": "Impossible de récupérer les informations de paiement.",
-            "addPaymentMethodSuccess": "La carte a été ajoutée avec succès.",
+            "savedCard": "Cartes enregistrées",
```

Comment:
missing translations `setDefault` and `delete`
![image](https://github.com/user-attachments/assets/b200d54f-686a-4055-bd92-7e3645bbff63)

Reply:
done